CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2.40.0-rc.0 / 2022-11-03
+
+This release introduces an experimental, native way of representing and storing histograms.
+It can be enabled in Prometheus via `--enable-feature=native-histograms` to accept native histograms.
+Use the `main` branch of `client_golang` to instrument your application with native histograms (this will change for the final release when v1.14.0 of client_golang will be out).
```

Comment:
This messaging is temporary for the release candidate. For the final release, (1) v1.14.0 of client_golang will be out with native histogram support, which I will mention here (2) I will be writing a blog post for prometheus.io blog introducing native histograms and explaining how to use it, and I will link that here as well.

Reply:
You also have to change an option in your instrumentation.

In the way it is worded now, it sounds a bit like all histograms will change to native histograms the moment you update client_golang and set the feature flag.

Reply:
@beorn7 how does it sound now?